src/main/java/ConsoleIO.java

```diff
@@ -0,0 +1,21 @@
+import java.io.*;
+
+public class ConsoleIO {
+    private BufferedReader inputStream;
+    private OutputStream outputStream;
+    private PrintStream printStream;
+
+    ConsoleIO(InputStream inputStream, OutputStream outputStream) {
+        this.inputStream = new BufferedReader(new InputStreamReader(inputStream));
+        this.outputStream = outputStream;
```

Comment:
Do you need this line? Maybe I'm missing something here but I don't see that you're using `this.outputStream` anywhere.